cypress/integration/events_test.spec.ts

```diff
@@ -1,107 +1,393 @@
+      "CODE_TO_TOKEN",
+      "CODE_TO_TOKEN_ERROR",
+      "LOGIN_ERROR",
+      "LOGIN",
+    ];
+
+    beforeEach(() => {
+      keycloakBefore();
+      loginPage.logIn();
+      sidebarPage.goToEvents();
+    });
+
+    it.skip("Check search dropdown display", () => {
+      sidebarPage.goToRealmSettings();
+      //realmSettingsPage.goToEventsTab().goToUserEventsSettingsSubTab().turnOffSaveEventsSwitch();
 
       masthead.signOut();
       loginPage.logIn();
 
       sidebarPage.goToEvents();
-
-      userEventsTab.shouldDisplay();
+      eventsPage.tabUtils().checkIsCurrentTab(EventsTab.UserEvents);
     });
 
     it("Check user events search form fields display", () => {
-      userEventsTab.shouldHaveFormFields();
+      userEventsTab
+        .openSearchUserEventDropdownMenu()
+        .assertUserSearchDropdownMenuHasLabels();
     });
 
     it("Check event type dropdown options exist", () => {
-      userEventsTab.shouldHaveEventTypeOptions();
+      userEventsTab
+        .openSearchUserEventDropdownMenu()
+        .openEventTypeSelectMenu()
+        .clickEventTypeSelectItem(eventTypes[0])
+        .clickEventTypeSelectItem(eventTypes[1])
+        .clickEventTypeSelectItem(eventTypes[2])
+        .clickEventTypeSelectItem(eventTypes[3])
+        .closeEventTypeSelectMenu();
     });
 
     it("Check `search events` button disabled by default", () => {
-      userEventsTab.shouldHaveSearchBtnDisabled();
+      userEventsTab
+        .openSearchUserEventDropdownMenu()
+        .assertSearchEventBtnIsEnabled(false);
     });
 
-    it.skip("Check user events search and removal work", () => {
-      userEventsTab.shouldDoSearchAndRemoveChips();
+    it("Check user events search and removal work", () => {
+      userEventsTab
+        .searchUserEventByEventType([eventTypes[0]])
+        .assertEventTypeChipGroupItemExist(eventTypes[0], true)
+        .assertEventTypeChipGroupItemExist(eventTypes[1], false)
+        .assertEventTypeChipGroupItemExist(eventTypes[2], false)
+        .assertEventTypeChipGroupItemExist(eventTypes[3], false)
+        .assertNoSearchResultsExist(true)
+        .removeEventTypeChipGroupItem(eventTypes[0])
+        .assertEventTypeChipGroupExist(false);
     });
 
     it("Check for no events logged", () => {
-      userEventsTab.shouldDoNoResultsSearch();
+      userEventsTab
+        .searchUserEventByUserId("test")
+        .assertNoSearchResultsExist(true);
     });
 
     it("Check `search events` button enabled", () => {
-      userEventsTab.shouldHaveSearchBtnEnabled();
+      userEventsTab
+        .openSearchUserEventDropdownMenu()
+        .assertSearchEventBtnIsEnabled(false)
+        .typeUserId("11111")
+        .assertSearchEventBtnIsEnabled(true);
+    });
+
+    it("Search by user ID", () => {
+      adminClient.loginUser(
+        eventsTestUser.userRepresentation.username,
+        eventsTestUser.userRepresentation.credentials[0].value,
+        eventsTestUserClientId
+      );
```

Comment:
Since this is a Promise it needs to be wrapped in [`cy.wrap()`](https://docs.cypress.io/api/commands/wrap#Promises).
```suggestion
      cy.wrap(null).then(() =>
        adminClient.loginUser(
          eventsTestUser.userRepresentation.username,
          eventsTestUser.userRepresentation.credentials[0].value,
          eventsTestUserClientId
        )
      );
```

Reply:
seems this test now is failing locally

Reply:
fixed test, it wasn't affected by this change